EasyDisenchant.lua

```diff
@@ -14,9 +14,43 @@ local _M = {
 	buttonRenderingCache = {},
 };
 
+EasyDisenchantBlacklist = {};
+
 BINDING_HEADER_EASY_DISENCHANT = _M.addonName;
 BINDING_NAME_EASY_DISENCHANT_OPEN = SHOW;
 
+local IsBlacklisted = function(itemLink)
+    for i, BLitemLink in pairs(EasyDisenchantBlacklist) do
```

Comment:
Core Lua functions like `pairs` should ideally have a local reference as it's faster access than the global scope. However, in my refactoring this was replaced with basic table iteration, and then later itemID key-based iteration.